src/plugins/intel_npu/src/plugin/src/backends.cpp

```diff
@@ -198,11 +198,6 @@ void NPUBackends::registerOptions(OptionsDesc& options) const {
     }
 }
 
-// TODO config should be also specified to backends, to allow use logging in devices and all levels below
-void NPUBackends::setup(const Config& config) {
-    _logger.setLevel(config.get<LOG_LEVEL>());
-}
```

Comment:
this part should not be removed.

Reply:
one case, to use set_property() to set log level and global log is none.

use set_property() set debug level(gloabl), all backend part will not be print because their origin log is no.

compile_model pass will cover global log. backend log will print by their log.  (getdeivce log)

seems need call backendsetup.

Are zerodevice and zeroinitstructholder(only construtor) contian any logs?

need add getDevice()  a updatesetup.
zerobackend, zerodevice  need a setup.